BackofficeBundle/Resources/public/less/smartadminPatches/widget.less

```diff
@@ -1,9 +1,27 @@
 .js-widget-title {
     margin-left: 10px;
+    display: block !important;
```

Comment:
this property isn't necessary, you should use it because in the template the div with class  .js-widget-title, has also the class display inline